DockerfileMakePackage

```diff
@@ -54,6 +55,7 @@ RUN find /ovms_release/lib/ -iname '*.so*' -exec patchelf --debug --set-rpath '$
 WORKDIR /ovms
 RUN cp -v /ovms/release_files/LICENSE /ovms_release/
 RUN cp -rv /ovms/release_files/thirdparty-licenses /ovms_release/
+RUN cp -v /ovms/src/test/SampleCustomLoader/libsampleloader.so /ovms_release/custom_loader/
```

Comment:
Why should we have custom loader example library in release files?

Reply:
Will change this along with unit testcases.